datasource/models/wikidata/wikidata.py

```diff
@@ -1,5 +1,8 @@
 import traceback
```

Comment:
sorry - i should have noticed this before, but would it be possible to migrate wikidata.py to the top of datasource/model and then move the other files into datasource/local/wikidata? That's the pattern the other datasources are following.

If you feel strongly about it, it would also be possible to move each of the datasources into their own subfolder in datasource/model, but I'd like to keep it consistent

Reply:
done